src/pages/using-spark/components/alert.mdx

```diff
@@ -19,24 +19,21 @@ provide feedback to user action.
 
 ### Usage
 
-Alert is used when you need to give
-information to the user or feedback
-after an action. For example, to show
-that a form submission was successful
-or to let the user know that their attention
-is needed on another section of the site.
-Alert may have timed visibility (10 seconds)
-and/or be dismissed by the user by clicking
-or tapping the Close Icon. 
+Use to direct attention to notable information, or to give feedback after an action.
+
+Example use cases:
+- Show an alert after successful form submission.
+- Direct to a part of the page that needs attention.
+- Notify users of critical information.
+- Inform a user with a dismissible Alert.
 
 ### Guidelines
 
-- Alert must appear above the main content
-of the page, but below the Masthead.
-- The message text of Alert should be
-clear and concise. 
-- Alert should not be used at the top of
-sidebars or other narrow layout elements. 
+- Place above the main content of the page, but below the Masthead.
+- Use a clear and concise message that communicates one piece of information per alert.
+- The message can contain a call to action through a link. Do not use buttons.
+- Upon choosing an Alert variant, decide how its dismissed. You can choose to dismiss after 10 seconds and/or by clicking the Close Icon.
```

Comment:
The first line feels extraneous. Also, is the 10 second thing built in to the component or something the app team has to implement?

Reply:
We've updated the documentation to provide clarity and made it more general (we don't have actual code that specifies the 10 seconds).